associations.rb

```diff
@@ -47,25 +57,42 @@ def generate_migrations
 
 class Hotel < ActiveRecord::Base
   #insert our associations here
- 
+  has_many :rooms
+
+  def bookings
+    Booking.where(room_id: rooms.map(&:id))
```

Comment:
Check out ["has_many :through" associations](http://guides.rubyonrails.org/association_basics.html#the-has-many-through-association).